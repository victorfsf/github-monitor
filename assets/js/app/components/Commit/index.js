import Commit from './Commit';

export default Commit;
