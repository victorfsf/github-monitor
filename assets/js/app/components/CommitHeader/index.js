import CommitHeader from './CommitHeader';

export default CommitHeader;
