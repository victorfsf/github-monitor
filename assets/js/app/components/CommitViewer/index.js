import CommitViewer from './CommitViewer';

export default CommitViewer;
