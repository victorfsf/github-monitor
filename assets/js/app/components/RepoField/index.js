import RepoField from './RepoField';

export default RepoField;
