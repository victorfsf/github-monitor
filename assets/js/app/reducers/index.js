import { combineReducers } from 'redux';

// TODO: write actual reducers
const mockedReducer = (state, action) => ({ mock: action });

const rootReducer = combineReducers({
  mockedReducer,
});

export default rootReducer;
