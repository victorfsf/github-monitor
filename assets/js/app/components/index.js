import Box from './Box';
import CommitViewer from './CommitViewer';
import Container from './Container';
import LoginForm from './LoginForm';
import RepoField from './RepoField';
import Spinner from './Spinner';
import Pagination from './Pagination';
import NotFound from './NotFound';

export {
  Box,
  CommitViewer,
  Container,
  LoginForm,
  RepoField,
  Spinner,
  Pagination,
  NotFound,
};
