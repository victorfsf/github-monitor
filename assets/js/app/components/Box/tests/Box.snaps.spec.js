import React from 'react';
import renderer from 'react-test-renderer';
import { Box } from 'app/components';


describe('Box', () => {
  let Component;
  let tree;

  test('does not render without children', () => {
    global.console = {
      error: jest.fn(),
    };
    renderer.create(<Box />);
    expect(global.console.error).toHaveBeenCalledTimes(1);
  });

  test('renders', () => {
    Component = renderer.create(<Box><div>Child</div></Box>);
    tree = Component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
