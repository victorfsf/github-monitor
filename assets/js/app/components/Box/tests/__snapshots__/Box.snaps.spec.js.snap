// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders 1`] = `
<div
  className="my-3 p-3 bg-white rounded box-border"
>
  <div>
    Child
  </div>
</div>
`;
